plugin-server/src/worker/vm/upgrades/historical-export/export-historical-events-v2.ts

```diff
@@ -119,10 +119,12 @@ type OffsetParams = Pick<ExportHistoricalEventsJobPayload, 'timestampCursor' | '
 export interface ExportHistoricalEventsUIPayload {
     dateRange: [string, string]
     parallelism?: number
+    // API-generated token
+    $job_id?: string
```

Comment:
nit/q: unsure why the $ - is it because it's not user-set?

(not very important)